controller/xt_smartrouting/impl.go

```diff
@@ -28,7 +30,8 @@ const (
 /**
 The smartrouting strategy relies purely on maninpulating costs and lets the smart routing algorithm pick the terminator.
 It increases costs by a small amount when a new session uses the terminator and drops it back down when the session
-finishes. It also increases the cost whenever a dial fails.
+finishes. It also increases the cost whenever a dial fails and decreases it whenever a dial succeeds. Dial successes
+will only reduce costs by the amount that failures have previously increased it.
```

Comment:
Corner case that we probably don't need to care about right this second, but a terminator with intermittent failures will likely look the same cost-wise as one without any.

Reply:
Yeah, figuring out how to account for that kind of thing is going to be fun. We'll have to see what testing and production scenarios turn up.

Reply:
It'll likely end up wanting some kind of short term memory. We'll figure it out.